fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/VersionedSchemaObject.java

```diff
@@ -6,6 +6,9 @@
 
 package com.ibm.fhir.database.utils.model;
 
+import java.util.Arrays;
+import java.util.List;
+
```

Comment:
The Title of the Java class makes me realize that we don't VERSION Indices.  I think we should (we have altered the signatures of a few of them without renaming them - drop-add)

Reply:
should we open it as a future task or you'd like to see that change as part of this PR?

Reply:
Let's open a future task.